### PR TITLE
Add new far auto for amp side

### PR DESCRIPTION
**Pull request by MatthewSC38**
It does like 3 notes and is untested, would like confirmation in the flip to the red side and that the commands are correct

### CODE REVIEW

**Review by aneeshaguna — APPROVED** [review]

File: src/autos/Gollum'sGoofyBlueQuest.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "version": 1.18,
+    "waypoints": [
+        {
+            "x": 0.7356774792774942,
+            "y": 6.859974617836561,
+            "r": -120,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyBlueQuest.json]:
```suggestion
            "r": 240,
```

File: src/autos/Gollum'sGoofyBlueQuest.json [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 52,
+            "c": "s",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 2.9030044503750982,
+            "y": 6.997899017805079,
+            "r": 0,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyBlueQuest.json]:
```suggestion
            "r": 180,
```

File: src/autos/Gollum'sGoofyBlueQuest.json [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 20,
+            "c": "G",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 0.7356774792774942,
+            "y": 6.859974617836561,
+            "r": -120,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyBlueQuest.json]:
```suggestion
            "r": 240,
```

File: src/autos/Gollum'sGoofyBlueQuest.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 52,
+            "c": "Sg",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 8.293573272885983,
+            "y": 7.452434156628877,
+            "r": 0,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyBlueQuest.json]:
```suggestion
            "r": 180,
```

File: src/autos/Gollum'sGoofyBlueQuest.json [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 20,
+            "c": "G",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 0.7490577833268346,
+            "y": 6.877645652298347,
+            "r": -120,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyBlueQuest.json]:
```suggestion
            "r": 240,
```

File: src/autos/Gollum'sGoofyQuest.json [hunk 6]
```diff
@@ -0,0 +1,50 @@
+{
+    "version": 1.18,
+    "waypoints": [
+        {
+            "x": 15.8545865207 ,
+            "y": 6.859974617836561,
+            "r": -60,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyQuest.json]:
```suggestion
            "r": 120,
```

File: src/autos/Gollum'sGoofyQuest.json [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 52,
+            "c": "s",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 13.6872595496,
+            "y": 6.997899017805079,
+            "r": 0,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyQuest.json]:
```suggestion
            "r": 180,
```

File: src/autos/Gollum'sGoofyQuest.json [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 20,
+            "c": "G",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 15.8545865207,
+            "y": 6.859974617836561,
+            "r": -60,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyQuest.json]:
```suggestion
            "r": 120,
```

File: src/autos/Gollum'sGoofyQuest.json [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 52,
+            "c": "Sg",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 8.29669072711,
+            "y": 7.452434156628877,
+            "r": 0,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyQuest.json]:
```suggestion
            "r": 180,
```

File: src/autos/Gollum'sGoofyQuest.json [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+            "p": 20,
+            "c": "G",
+            "v": 3,
+            "passthrough": 0
+        },
+        {
+            "x": 15.8545865207,
+            "y": 6.877645652298347,
+            "r": -60,
```
Comment (Omnikar) [inline src/autos/Gollum'sGoofyQuest.json]:
```suggestion
            "r": 120,
```

**Review by Omnikar — CHANGES_REQUESTED** [review]
Please also change all of the passthroughs to bools

File: src/autos/Gollum'sGoofyQuest.json (hunk 8 above)
Comment (MatthewSC38) [reply src/autos/Gollum'sGoofyQuest.json]: It should be 300 right?

**Pull request closed without merging**